### ISSUE TITLE
Fix or handle `Operation timed out` on connect

### ISSUE THREAD

**Issue by davidfoliveira** [davidfoliveira/homeswitch-api#2]
FIX this:
```
2020-11-10 20:44:42.755958Z: [WARN] Socket (192.168.10.179:6668) error occurred: <class 'socket.error'> [Errno 60] Operation timed out <traceback object at 0x1053ef710>
2020-11-10 20:44:42.758109Z: [ERRO] Socket error on device's bf5d0abdb1e6 connection: ['Traceback (most recent call last):\n', '  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/asyncore.py", line 83, in read\n    obj.handle_read_event()\n', '  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/asyncore.py", line 446, in handle_read_event\n    self.handle_connect_event()\n', '  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/asyncore.py", line 454, in handle_connect_event\n    raise socket.error(err, _strerror(err))\n', 'error: [Errno 60] Operation timed out\n']
```

Possibly related to device instability / device down.

**Comment by davidfoliveira** [comment]
Should be addressed with #11

**Comment by davidfoliveira** [comment]
I think this is solved by other asyncore stability related changes.
